packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -112,3 +112,48 @@ Template locking allows locking the `InnerBlocks` area for the current template.
 If locking is not set in an `InnerBlocks` area: the locking of the parent `InnerBlocks` area is used.
 
 If the block is a top level block: the locking of the Custom Post Type is used.
+
+### `renderAppender`
+* **Type:** `Function`
+* **Default:** - `null`. By default the `<DefaultBlockAppender>` component is used which automatically inserts whichever block is configured as the default block via `wp.blocks.getDefaultBlockName` (typically `paragraph`).
+
+Determines the placeholder behaviour when the Block is initially inserted. You may pass a render function to provide your own placeholder as required.
+
+#### Notes
+* For convenience two predefined appender components are exposed on `InnerBlocks` which can be consumed within the render function:
+	- `<InnerBlocks.ButtonBlockAppender />` -  display a `+` (plus) icon inside a box which fills the Block preview. No default Block is inserted. Clicking the appender reveals the Block picker UI.
+	- `<InnerBlocks.DefaultBlockAppender />` - provides the default "auto-insert" functionality described above under `default`.
+	- `<InnerBlocks.HideWhenChildBlocks />` - will only render `children` if the instance of `InnerBlocks` has no child blocks (typically only at point of first insertion). Once a child Block is added the `children` will not be rendered (see example usage below)
```

Comment:
Should this be a component or a prop passed to the renderProp?

`renderAppender={ ({ hasChildren }) => hasChildren && <InnerBlocks.ButtonBlockAppender /> }`

Reply:
I'm proposing adding this to a separate PR, as it doesn't seem critical at the moment, and could do with some separate exploration.